enterprise/views.py

```diff
@@ -416,6 +418,26 @@ def get_context_from_db(self, consent_page, platform_name, item, context):
         }
         return context_data
 
+    def is_course_run_id(self, course_id):
+        """
+        Returns True if the course_id is in the correct format of a course_run_id, false otherwise.
+
+        Arguments:
+            course_id (str): The course_key or course run id
+
+        Returns:
+            (Boolean): True or False
+        """
+        try:
+            # Check if we have a course ID or a course run ID
+            CourseKey.from_string(course_id)
+        except InvalidKeyError:
+            # The ID we have is for a course instead of a course run
+            return False
+        else:
```

Comment:
no need to put else after the return statement.
```
except InvalidKeyError:
            # The ID we have is for a course instead of a course run
            return False
# If here, the course_id is a course_run_id
return True
```